Add "Fix an issue" to the hackathon ideas (#16)
MDAnalysis/UGM2023: hackathon/fix-an-issue/README.md

# Fixing an issue in the MDAnalysis core library

With over [400 issues to deal with](https://github.com/MDAnalysis/mdanalysis/issues)
we are always looking for new and old contributors to help enhance
and maintain the [MDAnalysis core library](https://github.com/MDAnalysis/mdanalysis).

## Team

> The MDAnalysis core developers team (@IAlibay, @lilyminium, @fiona-nauthon,

Author: I just added everyone for now, but maybe that's not a good idea

Reviewer: Having names here is good. We'll be helping anyway.

@orbeckst, @richardjgowers, @micaela-matta) will help you along your way
towards contributing a code change to MDAnalysis

## Getting started

In order to contribute to MDAnalysis, you will need to set up a
development environment, make a fork of the MDAnalysis codebase
and then contribute the code back using a [Pull Request]().

Our [User Guide]() has a handy [contributors guide]() which you can
follow for all the information you need to achieve this. Feel free
to ask any members of the team for help if you encounter any issues
gettting started.

> ## Choose an issue

Author: Do we need a section on the review process?

Author: Added

We have plenty of issues available for you to work on in our
[Issue Tracker](). For first time contributors, we encourage
you to look at our list of [easy bugs]() to fix. Have a chat with us,
to see which one would suit you best! When you found something, please
let us know in the issue that you are working on it so that other
attendees do not work on the same thing!

> Our recommended pick is the [doctest MEGA-issue]() which involves

Author: This would be significantly easier for us to review.

# Fixing an issue in the MDAnalysis core library

With over [400 issues to deal with](https://github.com/MDAnalysis/mdanalysis/issues)
we are always looking for new and old contributors to help enhance
and maintain the [MDAnalysis core library](https://github.com/MDAnalysis/mdanalysis).

## Team

> The MDAnalysis core developers team (@IAlibay, @lilyminium, @fiona-nauthon,

Reviewer: fix
```suggestion
The MDAnalysis core developers team (@IAlibay, @lilyminium, @fiona-naughton,
```

@orbeckst, @richardjgowers, @micaela-matta) will help you along your way
towards contributing a code change to MDAnalysis

## Getting started

In order to contribute to MDAnalysis, you will need to set up a
development environment, make a fork of the MDAnalysis codebase
and then contribute the code back using a [Pull Request]().

Our [User Guide]() has a handy [contributors guide]() which you can
follow for all the information you need to achieve this. Feel free
to ask any members of the team for help if you encounter any issues

> gettting started.

Reviewer 2: ```suggestion
getting started.
```

## Choose an issue

We have plenty of issues available for you to work on in our
[Issue Tracker](). For first time contributors, we encourage
you to look at our list of [easy bugs]() to fix. Have a chat with us,

> to see which one would suit you best! When you found something, please

Reviewer 2: ```suggestion
to see which one would suit you best! When you find something, please
```

let us know in the issue that you are working on it so that other
attendees do not work on the same thing!

Our recommended pick is the [doctest MEGA-issue]() which involves
fixing our documentation so that code examples are self-contained

> and should run on their own if you tried them out! Here multiple

Reviewer 2: ```suggestion
and should run on their own if you try them out! Here multiple
```

attendees can work on this issue, just let us know which files you
are planning to work on.

## Make a Pull Request!

Once you have made the suitable code change to fix the issue,
[open a Pull Request]() and we will review your code change
and work with you towards getting your contribution merged.

This can sometimes be a slow process, especially when the
code change is large and there are many Pull Requests. If you

> wish to help, you are welcome to review other's Pull Requests

Reviewer 2: ```suggestion
wish to help, you are welcome to review other people's Pull Requests
```
